### ISSUE TITLE
Audit progress: show if a jurisdiction has logged in

### ISSUE THREAD

**Issue by jonahkagan** [votingworks/arlo#1511]
Add an additional status to the progress screen (table and map) to show if any jurisdiction manager from a jurisdiction has logged in yet. This will help audit admins differentiate jurisdictions that haven't touched Arlo at all from jurisdictions that have logged in but haven't uploaded a ballot manifest yet, so they know who to nag :)

**Comment by jonahkagan** [comment]
Closing in favor of: https://github.com/votingworks/arlo/issues/1512

**Comment by ginvdr** [comment]
@jonahkagan I think this one: https://github.com/votingworks/arlo/issues/1512 accomplishes the same thing.

**Comment by jonahkagan** [comment]
Yep, meant to close this one but somehow didn't click the button
